pkg/platform/authentication/auth.go

```diff
@@ -44,12 +44,13 @@ type Auth struct {
 type Configurable interface {
 	Set(string, interface{}) error
 	GetString(string) string
+	Close() error
 }
 
-// Get returns a cached version of Auth
-func Get() *Auth {
+// LegacyGet returns a cached version of Auth
+func LegacyGet() *Auth {
 	if persist == nil {
-		cfg, err := config.Get()
+		cfg, err := config.New()
```

Comment:
Hmm... Technically, creating a config instance with `New` means that we create a new `singlethread` instance too, no?  And that would mean that a process could have two `singlethread`s...

Reply:
I have a rationale for this;

Long term there should really only be one `config.New()` per process, though exceptions to that rule are of course possible. BUT when there are multiple `config.New()` calls it's unlikely that they'd be dealing with the same config keys.

We could also have a single singlethread instance, but I was trying to avoid getting THAT far into the weeds.

For now I consider this to be a non-issue, but it's something we should keep in mind when we design the reusable system to set up commands, which would include the primer stuff that would be necessary to properly address these types of use cases.

Reply:
Okay.  That is fine with me.